src/sentry/static/sentry/app/views/settings/types.ts

```diff
@@ -0,0 +1,52 @@
+import {Organization, Project} from 'app/types';
+
+export type NavigationProps = {
+  id?: string;
+  organization?: Organization;
+  project?: Project;
+  access?: Set<string>;
```

Comment:
@EvanPurkhiser Isn't there a set list of access strings? Could we do better than `string`?

Reply:
I don't think we have these as a defined type yet.

Reply:
@EvanPurkhiser Would it require a lot of code changes to add?

Reply:
Unsure, but it's something I would rather do separate from this I think, since I'd prefer to do it consistently.